apps/daffio/src/scss/styles.scss

```diff
@@ -4,6 +4,23 @@
 
 // Imports
 @import 'daff-typography';
+@import 'daff-theme';
+
+@import "../app/content/homepage/component/homepage-theme.scss";
+@import "../app/content/pwa/components/feature-comparison/feature-comparison-theme";
+@import "../app/content/pwa/pages/pwa/pwa-theme";
+@import "../app/core/footer/simple-footer/simple-footer-theme";
+@import "../app/core/footer/sub-footer/sub-footer-theme";
+
+@import "theme";
+
+@include daff-theme($theme);
+
+@include daffio-homepage-theme($theme);
+@include daffio-pwa-theme($theme);
+@include daffio-simple-footer-theme($theme);
+@include daffio-sub-footer-theme($theme);
+@include daffio-feature-comparison-theme($theme);
```

Comment:
why are these imported in a global styling file? Wouldn't it be better to have them in each of their component scss files? Like the @include daffio-pwa-theme($theme); goes in the pwa.component.scss file

Reply:
I agree.

Reply:
@lderrickable the idea here is that these mixins include SCSS. We'd face two issues if we tried to include them in the component file rather than at the global level:

1. Specificity, components that are ViewEncapsulated get `[nghost]` on them which would
2. Ability to do multi-theme without having to specify classes all over the place, we can quickly centralize them at the top of the app and apply a single class globally.